src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,77 @@
+# social:
+#   google:
+#     title: A title for Google search card.
+#     description: A description for Google search card.
+#     thumbnail: google_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   facebook:
+#     title: A title for Facebook card.
+#     description: A description for Facebook card.
+#     thumbnail: facebook_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   twitter:
+#     title: A title for Twitter card.
+#     description: A description for Twitter card.
+#     thumbnail: twitter_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+If you've installed a Progressive Web App on a recent Android phone, you might notice the icon shows up with a white background. {{image}}. Android Oreo introduced adaptive icons, which display app icons in a variety of shapes across different device models. Icons that don't follow this new format are given white backgrounds.
+
+Maskable icons are a new icon format that let your Progressive Web App use adaptive icons. If you supply a maskable icon, your icon can fill up the entire shape and look great on all Android devices. Firefox and Chrome have recently added support for this new format, and you can adopt it in your apps.
+
+## Are my current icons ready?
+
+Since maskable icons need to support a variety of shapes, you supply an opaque image with some padding that the browser can later crop into the correct shape and size. The important parts of your icon, such as your logo, should be within the middle 80% of the icon. The outer 10% edge may be cropped.
+
+You can check which parts of your icons land within the safe zone with Chrome DevTools. With your Progressive Web App open, open DevTools and navigate to the "Application" panel. In the "Icons" section, you can choose to "Show only the minimum safe area for maskable icons". {{image}}. Your maskable icons will be trimmed so that only the safe area is visible. If your logo is visible within this area, you're good to go.
+
+If you want to preview your maskable icon in other shapes it may appear in on Android, I've created a tool called [maskable.app](https://maskable.app/). Upload an icon and maskable.app will let you try out various shapes and sizes, and you can share the preview with others on your team. {{image}}.
+
+
+## How do I adopt maskable icons?
+
+If you want to create a maskable icon based off your existing transparent icon, you can use the [maskable.app editor](https://maskable.app/editor). Upload your icon, adjust the color and size, then export the image. {{image}}.
+
+Once you've created a maskable icon image and tested it out in DevTools, you'll need to update your Web App Manifest to point to the new assets. The Web App Manifest provides information about your web app in a JSON file, and includes an `icons` array. {{code}}.
```

Comment:
Opportunity to link to https://developers.google.com/web/fundamentals/web-app-manifest. cc @PaulKinlan @gauntface, are there any plans to port this to web.dev?